frontend/test/__support__/e2e/helpers/e2e-cloud-helpers.js

```diff
@@ -0,0 +1,10 @@
+export function setupMetabaseCloud() {
+  const path = "/api/setting/site-url";
+  cy.request("GET", path).then(response => {
+    const siteUrlSetting = response.body;
+    cy.request("PUT", path, {
+      ...siteUrlSetting,
+      value: "https://CYPRESSTESTENVIRONMENT.metabaseapp.com",
+    });
+  });
+}
```

Comment:
Hey @nemanjaglumac, I tried to come up with a helper that would allow running E2E tests in a "cloud context". Looks hacky, maybe you have a better idea of how to achieve the same? There is a [comment saying we're going to have an environment variable defining if Metabase is hosted](https://github.com/metabase/metabase/blob/b5d48172472c2bd32157e1df088b6504a4f545a0/frontend/src/metabase/lib/settings.js#L90), so we can update the helper implementation later maybe?

Reply:
There's a magic trick for this :)
```javascript
mockSessionProperty(
  "site-url",
  "https://CYPRESSTESTENVIRONMENT.metabaseapp.com",
);
```

Reply:
Real magic 🪄 Thank you!

Reply:
UPDATE: looks like it doesn't work properly with `mockSessionProperty`, so keeping that as is for now